### DISCUSSION TITLE
Set Width of "Label" Column in Field Group

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5451] [Ideas]
Be able to set a fixed width for the label column on the Field Group component
, to have uniformity across forms where field names are different.

**Comment by melohagan** [comment]
Hey @poirazis 

The label widths look consistent for me. Would you mind providing a screenshot to show the mis-alignment?

**Comment by poirazis** [comment]
Sure
I have Form -> Container -> Section (two columns) - 2 x Field Groups
You can see the misalignment in the mobile view.


<img width="982" alt="Screenshot 2022-04-19 at 7 37 21 PM" src="https://user-images.githubusercontent.com/80512170/164053139-2f6c241f-5bbe-4172-9a36-37fe89e4ca3a.png">
<img width="480" alt="Screenshot 2022-04-19 at 7 37 32 PM" src="https://user-images.githubusercontent.com/80512170/164053174-c7a558e2-66f1-4580-80f8-27694dbac4bb.png">

**Reply by joebudi** [reply]
Hi @poirazis 

What if you tried:
Form > Field group (labels above) > Section (2 column) > Fields.

Another option might be to use conditional UI to update the label position on mobile:
Field group > conditional UI > (image below)
![CleanShot 2022-04-19 at 17 50 08](https://user-images.githubusercontent.com/49767913/164055026-b252d0e3-4e4f-4d68-bf90-b47013cfa8a2.png)

**Reply by poirazis** [reply]
Yes , with the labels above it works ! 
Thank you, good sir ! 

My idea was if we could set a min width on the field label, it would enable more design capabilities !

**Reply by joebudi** [reply]
haha I've been there before hence why I knew the answer.

**Comment by poirazis** [comment]
Hey @joebudi :)
Sorry to be a buster, but i think i identified an issue with your approach.

In the below example, I have Form > Field group (labels above) > Section (2 column) > Fields.

This introduces another "issue", the Vertical space you can see in the screenshot for the fields 
![VertAlign](https://user-images.githubusercontent.com/80512170/164479201-e170550d-6531-4aa5-87c3-2ec2eb0e2c85.png)

**Reply by joebudi** [reply]
Hey. You can revert back to your original way, but just conditionally update the label position.

Desktop - side
Mobile - above

I think this should work.
